parent/pom.xml

```diff
@@ -954,6 +963,7 @@
                             <ignorePackages>
                                 <package>org.apache.ignite.internal.sql.engine</package>
                                 <package>org.apache.ignite.internal.generated</package>
+                                <package>org.apache.ignite.rest.client</package>
```

Comment:
Do I understand correctly that this package contains clients generated by Swagger? If yes, please add a comment, if no - why did you add it here?

Reply:
Yes, you are right. I'll add a comment.